nearlib/dev.js

```diff
@@ -9,16 +9,23 @@ module.exports = {
     connect: async function(nodeUrl) {
         const studioConfig = await this.getConfig();
         const near = nearlib.Near.createDefaultConfig(nodeUrl || studioConfig.nodeUrl);
-        await this.getOrCreateDevUser();
+        await this.getOrCreateDevUser(near);
         return near;
-    }, 
-    getOrCreateDevUser: async function () {
+    },
+    getOrCreateDevUser: async function (near) {
         let tempUserAccountId = window.localStorage.getItem(localStorageAccountIdKey);
         if (tempUserAccountId) {
-            return tempUserAccountId;
+            // Make sure the user actually exists and recreate it if it doesn't
+            const accountLib = new nearlib.Account(near.nearClient);
+            try {
+                const accountLookupResult = await accountLib.viewAccount(tempUserAccountId);
+                return tempUserAccountId;
+            } catch (_) {
+                // Something went wrong! Recreate user by continuing the flow
```

Comment:
I think we should log error and this message to console, to avoid confusing when debugging

Reply:
done